pkgs/games/steam/fhsenv.nix

```diff
@@ -344,6 +344,10 @@ in buildFHSEnv rec {
       description = "Run commands in the same FHS environment that is used for Steam";
       mainProgram = "steam-run";
       name = "steam-run";
+      # steam-run itself is just a script that lives in nixpkgs (which is licensed under MIT).
+      # steam is a dependency and already unfree, so normal steam-run will not install without
+      # allowing unfree packages or appropriate `allowUnfreePredicate` rules.
+      license = lib.licenses.mit;
```

Comment:
This isn't technically necessary but it doesn't hurt to be explicit either.

Reply:
It just bugged me that I had to add `steam-run` to `allowUnfreePredicate` even though `steam-run` itself is not unfree.

Reply:
runCommand should be free by default but it's actually inheriting the meta from steam, so that's why we need to override here.